CMakeLists.txt

```diff
@@ -40,8 +40,9 @@ include(CheckCXXCompilerFlag)
 include(CheckLinkerFlag)
 include(CheckLibraryExists)
 include(CheckFunctionExists)
-
-cmake_policy(SET CMP0148 OLD)
+if (NOT("${CMAKE_VERSION}" VERSION_LESS 3.27)) # If CMake version >= 3.27
```

Comment:
```
if(POLICY CMP0148)
  cmake_policy(SET CMP0148 OLD) # https://cmake.org/cmake/help/latest/policy/CMP0148.html
endif()
```

It is a cleaner syntax IMHO [1]. 

1. https://cmake.org/cmake/help/latest/command/if.html#policy